packages/terra-progress-bar/docs/progress-bar-customized-colors.md

```diff
@@ -0,0 +1,80 @@
+## Progress Bar Customized Colors
+
+By defining more CSS, authors can control more of the `<ProgressBar />` component through the colorClass prop.
+
+```css
+.green-bar-grey-background-white-border {
+  background-color: #d3d4d5;
+  border-right: 1px solid #fff;
+  color: #8ccc62;
+
+  &[value] {
+    &::-webkit-progress-bar {
+      background-color: #d3d4d5;
+    }
+
+    &::-webkit-progress-value {
+      background-color: #8ccc62;
+      border-right: 1px solid #fff;
+    }
+
+    &::-ms-fill {
```

Comment:
same in this file

Reply:
Added here https://github.com/cerner/terra-core/pull/2211/commits/ca98f46aabe3c7c3ca17f511d43ec994085fa01c